script/Ciel/Day 14/14_01B_03_CIEL14B_5B.txt

```diff
@@ -897,13 +898,15 @@ Even though I do not dream―――
 -- Page 260, Offset 41006.
 -- その<夢|もしも>に、一生を費やして恋い焦がれる。
 // Man we are just pulling out all the themes today
+// Correct verb?
 I will spend my whole life yearning for that what-if.
 }
 [sha:f8becfe84eba2be6b4259898fa9c32f5ae739022]{
 -- Page 260, Offset 41007.
 -- 胸に咲く花のような、出会う事のなかった運命を。
 // Rouage fan?
 // https://www.youtube.com/watch?v=lhn0xch8QAs
+// What did he mean by this
 Yearning for a world in which we never met;
  one like a flower that blooms inside my breast.
```

Comment:
I don't get it agh
Well, I do feel like the second clause needs a little something, but it's just some small adjustments
>one like a flower blossoming within my breast.